docs/HowTo/Get-Started/Manage-Memory.md

```diff
@@ -11,6 +11,9 @@ We recommend setting the maximum heap size to 2GB if you have less than 8GB of R
 more RAM, but want to free up memory for other processes. This results in an approximate process
 size of 3.5GB (on Linux).
 
+If you have more than 8GB of RAM, we recommend setting the maximum heap size to approximately
+5.632GB.
+
```

Comment:
if you have 8GB of ram, XXX gb works well. 

If you have more than 8GB of ram, total memory - 4GB if the machine is only running a single instance of teku.
If the machine is used for multiple tasks, set java heap space to 6GB for teku, to limit it's memory usage to approximately 8GB.
that's my attempt, @ajsutton or @mbaxter would potentially have far more accurate answers...

Reply:
I'd say,

> If you have more than 8GB of RAM available, we recommend setting the maximum heap size to 5GB or more.

It's not an exact science, and half gigabytes are neither here nor there.

Reply:
Done.